### DISCUSSION TITLE
Github AE - Github Codespaces  support ?

### DISCUSSION THREAD

**Discussion by cliffordfajardo** [community/community#8484] [Codespaces] [answered]
Hi Github team,
I am a LinkedIn employee. At LinkedIn, we currently use Github.

Do you know when this Github Codespace feature will be available for Github AE accounts?

- https://docs.github.com/en/github-ae@latest/admin/overview/about-github-ae

Thanks

EDIT
- Github AE public roadmap: https://github.com/orgs/github/projects/4247/views/3?filterQuery=label%3Aae+

**Comment by asciimike** [comment, marked as answer]
@cliffordfajardo no current plans for GHAE support, but we'll re-evaluate once GHAE is more widely adopted.
